packages/ra-ui-materialui/src/layout/Loading.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import polyglotI18nProvider from 'ra-i18n-polyglot';
+import englishMessages from 'ra-language-english';
+import { I18nContextProvider, Resource, testDataProvider } from 'ra-core';
+
+import { AdminContext } from '../AdminContext';
+import { AdminUI } from '../AdminUI';
+import { Loading } from './Loading';
+
+export default {
+    title: 'ra-ui-materialui/layout/Loading',
+};
+
+const i18nProvider = polyglotI18nProvider(() => englishMessages, 'en');
+
+export const Basic = () => <Loading />;
+
+export const I18N = () => {
+    return (
+        <I18nContextProvider value={i18nProvider}>
+            <Loading />
+        </I18nContextProvider>
+    );
+};
+
+export const InBox = () => (
+    <div
+        style={{
+            position: 'absolute',
+            top: 25,
+            left: 40,
+            height: 250,
+            width: 400,
+            border: '1px solid gray',
+        }}
+    >
+        <Loading />
+    </div>
+);
+
+export const FullApp = () => (
```

Comment:
This one doesn't work as expected: when clicking on a menu item, the entire app reloads... I have to investigate

Reply:
Fixed: the bug appeared in the story because it didn't have an auth provider, which shouldn't happen in reality when using a resource function.